libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -678,33 +826,36 @@ class CreatorNode {
         }
 
         // if the content node returns an invalid auth token error, clear connection and reconnect
-        if (resp.data && resp.data.error && resp.data.error.includes('Invalid authentication token')) {
+        if (resp?.data?.error?.includes('Invalid authentication token')) {
           this.clearConnection()
           try {
             await this.ensureConnected()
           } catch (e) {
-            console.error(e.message)
+            console.error((e as Error).message)
           }
         }
 
-        await this._handleErrorHelper(e, axiosRequestObj.url, requestId)
+        await this._handleErrorHelper(error, axiosRequestObj.url, requestId)
       }
     }
-    return retry(async (bail, num) => {
-      return work()
-    }, {
-      // Retry function 3x
-      // 1st retry delay = 500ms, 2nd = 1500ms, 3rd...nth retry = 4000 ms (capped)
-      minTimeout: 500,
-      maxTimeout: 4000,
-      factor: 3,
-      retries: 3,
-      onRetry: (err, i) => {
-        if (err) {
-          console.log('makeRequest retry error: ', err)
+    return await retry(
```

Comment:
same Q here about how this throws

Reply:
ah so looked into this, and this is being auto-fixed by lint, this is the rule: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/return-await.md looks like the idea is that it helps with stach-track info, which makes sense. since it's autofixable my guess is that it shouldn't break semantics